engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -0,0 +1,384 @@
+/* ResidualVM - A 3D game interpreter
+ *
+ * ResidualVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "base_render_opengl3d.h"
+
+#include "../../../../../graphics/opengl/system_headers.h"
+#include "base_surface_opengl3d.h"
+#include "math/glmath.h"
+
+namespace Wintermute {
+BaseRenderer *makeOpenGL3DRenderer(BaseGame *inGame) {
+	return new BaseRenderOpenGL3D(inGame);
+}
+
+} // namespace Wintermute
+
+Wintermute::BaseRenderOpenGL3D::BaseRenderOpenGL3D(Wintermute::BaseGame *inGame)
+    : _spriteBatchMode(false) {
+}
+
+Wintermute::BaseRenderOpenGL3D::~BaseRenderOpenGL3D() {
+}
+
+Wintermute::BaseImage *Wintermute::BaseRenderOpenGL3D::takeScreenshot() {
+	return nullptr;
+}
+
+bool Wintermute::BaseRenderOpenGL3D::saveScreenShot(const Common::String &filename, int sizeX, int sizeY) {
+	return true;
```

Comment:
Seems to be some outstanding work here, consider adding something like
warning("Screenshot-functionality not yet implemented for OpenGL-renderer");

So that it's visible that there's stuff TODO here.